eng/pipelines/templates/steps/publish-cli.yml

```diff
@@ -18,6 +18,8 @@ parameters:
   SyndicatedAcrPassword: $(azdev-acr-syndicated-password)
   PublishBrewFormula: false
   PublishArm64Binaries: false
+  AllowPrerelease: false
```

Comment:
Just curious. This is always false because we use eng-scripts that require this value, but publish_cli is meant to always publish official, whereas `Publish_Integration`, a separate stage, is for publishing prerelease that never goes to official channels? It's hard to understand at a glance what this means.

Reply:
Indeed this is the result of this template getting really complex. Publishing should instead be split into separate jobs... https://github.com/Azure/azure-dev/issues/2418

Reply:
Also, the one place where `AllowPrerelease` is used is gated behind 

```yaml
  - ${{ if eq('true', parameters.PublishBrewFormula) }}:
```

In that case you'll need to specify `PublishBrewFormula: true` and that's only the case in release builds.